src/test/java/Homework19.java

```diff
@@ -0,0 +1,37 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework19 extends BaseTest{
+    @Test
+    public void deletePlaylist() throws InterruptedException{
+       //navigateToPage();
+provideEmail("varag@testpro.io");
+providePassword("te$t$tudent1");
+loginToKoel();
+Thread.sleep(2000);
+choosePlaylist();
+Thread.sleep(2000);
+xPlaylist();
+Thread.sleep(2000);
+
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)